projects/pgai/db/sql/idempotent/011-vectorizer-int.sql

```diff
@@ -967,13 +1197,22 @@ begin
     set local search_path = pg_catalog, pg_temp;
 
     -- if the conditions are right, create the vectorizer index
-    if ai._vectorizer_should_create_vector_index(_vec) then
+    if ai._vectorizer_should_create_vector_index(_vec) and _vec.config operator(pg_catalog.->) 'destination' operator(pg_catalog.->>) 'implementation' operator(pg_catalog.=) 'table' then
```

Comment:
nit: don't love that we have to call `ai._vectorizer_should_create_vector_index` twice: in the if and the elseif. Can we "cache" that in a variable?